src/plugins/intel_gpu/src/graph/reshape.cpp

```diff
@@ -23,8 +23,6 @@ layout reshape_inst::calc_output_layout(reshape_node const& node, kernel_impl_pa
     if (desc->output_shape.count() == 0) {
         if (desc->output_partial_shape.size() != 0) {
             return layout{desc->output_partial_shape, input_layout.data_type, input_layout.format};
-        } else {
-            OPENVINO_THROW("[GPU] Output shape is not provided");
```

Comment:
This is called for static shape? 
Then it should have output shape.. Could you elaborate detail about the target behavior?

Reply:
The previous node is StridedSlice, which has an empty output shape because its input is [1, 1] and shrink_mask is [1, 1].